deploy/040-controller.yaml

```diff
@@ -30,7 +30,6 @@ spec:
           image: quay.io/kubevirt/csi-driver:latest
           args:
             - "--endpoint=$(CSI_ENDPOINT)"
-            - "--namespace=kubevirt-csi-driver"
```

Comment:
this arg is no longer used and caused the csi driver to fail since it's an unknown cli arg now